### ISSUE TITLE
📎 Link Button – Usability Discussion

### ISSUE THREAD

**Issue by Jorre-student** [Obra-Studio/shadcn-ui-kit#28]
Hi! I wanted to start a discussion about the link button.

From how I interpret the WCAG, clickable text should have a clear signifier. With link buttons, I feel that this isn’t always the case, without an underline or an icon, it can sometimes be overlooked as clickable text.

Curious to hear your thoughts on the clarity and accessibility of the link button.

Maybe adding a note like we did with hovercards could be usefull?

**Comment by shadcn** [comment]
@Wolfr I probably missed it when I upgraded the site. We still have the link variant in the code. 

https://github.com/shadcn-ui/ui/blob/84d6c83bad10e995bd648d59403f402ce7728444/apps/v4/registry/new-york-v4/ui/button.tsx#L22

**Comment by Wolfr** [comment]
Cleaning old issues, closing this.

Thanks for your input @MaximCalluy !
